Makefile

```diff
@@ -326,6 +317,10 @@ endef
 
 install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
 	@$(MAKE) $(QUIET_MAKE) all
+	# Third-party code that is not controlled by Pkg should be installed into
+	# prefix/share/julia/site/VERSDIR (not prefix/share/julia/VERSDIR/site)
```

Comment:
The previous ellipsis achieved its goal of confusing the reader… The parentheses are balanced now.